release.sh

```diff
@@ -83,7 +83,7 @@ usage() {
 	echo "  -p curse-id      Set the project id used on CurseForge for localization and uploading. (Use 0 to unset the TOC value)" >&2
 	echo "  -w wowi-id       Set the addon id used on WoWInterface for uploading. (Use 0 to unset the TOC value)" >&2
 	echo "  -a wago-id       Set the project id used on Wago Addons for uploading. (Use 0 to unset the TOC value)" >&2
-	echo "  -g game-version  Set the game version to use for CurseForge uploading." >&2
+	echo "  -g game-version  Set the game version to use for CurseForge and Wago uploading (currently supports 'classic' and 'bc')." >&2
```

Comment:
-g also supports specificying an actual version directly (eg. -g 1.13.7), so the help update is not accurate